Folge_02/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "setup-react",
+  "version": "1.0.0",
+  "description": "Eine Hello-World Applikation mit React",
+  "main": "index.js",
+  "scripts": {
+    "start": "npx servor --reload",
+    "compile": "babel src --out-dir lib",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "David Losert",
+  "license": "MIT",
+  "devDependencies": {
+    "@babel/cli": "^7.8.4",
+    "@babel/core": "^7.9.6",
+    "@babel/preset-react": "^7.9.4"
```

Hier würde ich auf jeden Fall die Carets entfernen, das macht es sonst schwer(er) nachstellbar.

```suggestion
    "@babel/cli": "7.8.4",
    "@babel/core": "7.9.6",
    "@babel/preset-react": "7.9.4"
```